lib/grinder_sdk.dart

```diff
@@ -17,6 +18,17 @@ import 'src/utils.dart';
 
 bool _sdkOnPath;
 
+// Returns the common top-level directories which usually contain Dart source
+// and which actually exist in the current project.
+Set<String> get defaultSourceDirectories => [
```

Comment:
Thanks for pointing that out. I thought it was just a comment on another issue. 
`Directory` seems not to have `==`, `identical` seems a bit overkill, and I don't like that one returns realtive dirs, while the other returns absolute ones.
This looks a bit overcomplicated but it was the simplest I found
```
Set<Directory> get existingSourceDirs => Directory.current
    .listSync()
    .where((f) => f is Directory)
    .map((d) => new Directory(path.relative(d.path)))
    .where((d) => sourceDirs.any((sd) => sd.path == d.path))
    .toSet();
```